docs/001_develop/03_client-capabilities/013_toast-notifications/index.mdx

```diff
@@ -2,17 +2,450 @@
+    </div>
+  `,
+})
+export class MyElement extends GenesisElement {
+  triggerNotification() {
+    this.$emit( NotificationEvent.NOTIFICATION_EVENT,
+      {
+        title: 'Important message',
+        body: 'Lorem ipsum',
+        config: {
+          snackbar: {
+            confirmingActions: [{ label: 'Confirm', action: () => console.log('Lorem ipsum') }],
+            type: 'error',
+            autoClose: true
+          },
+        },
+      } as NotificationStructure,)
+  }
+}
+```
+</TabItem>
+<TabItem value="react">
+```typescript
+import { NotificationEvent, NotificationStructure } from '@genesislcap/foundation-notifications';
+import { useRef } from 'react';
+
+export function MyComponent() {
+  const ref = useRef();
+  const triggerNotification = (event) => {
+    const notificationPayload = {
+      title: 'Important message',
+      body: 'Lorem ipsum',
+      config: {
+        snackbar: {
+          confirmingActions: [{ label: 'Confirm', action: () => console.log('Lorem ipsum') }],
+          type: 'error',
+          autoClose: true
+        },
+      },
+    } as NotificationStructure;
+    ref.current.dispatchEvent(customEventFactory(NotificationEvent.NOTIFICATION_EVENT, notificationPayload))
+  };
+
+  return (
+    <rapid-button onClick={triggerNotification} ref={ref}>Show Notification</rapid-button>
+  );
+}
+```
+</TabItem>
+<TabItem value="angular">
+```typescript
+import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { NotificationEvent, NotificationStructure } from '@genesislcap/foundation-notifications';
+
+@Component({
+  selector: 'my-root',
+  template: `
+    <rapid-button
+      (click)="triggerNotification($event)"
+    >Show Notification</rapid-button>
+  `,
+  standalone: true,
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+  imports: [FormsModule],
+})
+export class AppComponent {
+  constructor(private el: ElementRef) {}
+  triggerNotification(event: Event) {
+    const notificationPayload = {
+      title: 'Important message',
+      body: 'Lorem ipsum',
+      config: {
+        snackbar: {
+          confirmingActions: [{ label: 'Confirm', action: () => console.log('Lorem ipsum') }],
+          type: 'error',
+          autoClose: true
+        },
+      },
+    } as NotificationStructure;
+    this.el.nativeElement.dispatchEvent(customEventFactory(NotificationEvent.NOTIFICATION_EVENT, notificationPayload));
+  }
+}
+```
+</TabItem>
+</Tabs>
+
+`customEventFactory` is a helper function to create events in the format the notification event expects. You can either import it from the PBC module if you're using it, or copy this function into your codebase.
+
+```typescript
+export function customEventFactory(type: string, detail?: any) {
+  return new CustomEvent(type, {
+      bubbles: true,
+      cancelable: true,
+      composed: true,
+      detail,
+  });
+}
+```
```

Comment:
You can import this snippet like this
[Import](https://github.com/genesiscommunitysuccess/docs/blob/660ebbb5a655c08de626f38f2eb0890b4ae981b1/docs/001_develop/03_client-capabilities/017_state-management/legacy-setup.mdx?plain=1#L23)
and
[use](https://github.com/genesiscommunitysuccess/docs/blob/660ebbb5a655c08de626f38f2eb0890b4ae981b1/docs/001_develop/03_client-capabilities/017_state-management/legacy-setup.mdx?plain=1#L264)